### ISSUE TITLE
error with countriesLow

### ISSUE THREAD

**Issue by mdsumner** [eblondel/cleangeo#7]
I get this in 0.1-0

library(rworldmap)
data(countriesLow)
 library(cleangeo)
x <- clgeo_Clean(countriesLow)
Error in SpatialPolygonsDataFrame(Sr = fixed.sp, data = as(sp, "data.frame")) : 
  row.names of data and Polygons IDs do not match

Thanks!  (I'll explore more when I have a chance).

**Comment by eblondel** [comment]
Thanks @mdsumner i will have a look later today and report to you asap

**Comment by eblondel** [comment]
Fixed.

Please check:

```{r}
library(rworldmap)
data(countriesLow)

#inspect input data
sp <- countriesLow
report <- clgeo_CollectionReport(sp)
clgeo_SummaryReport(report)

#fix the geometries & inspect again
sp.fixed <- clgeo_Clean(sp)
report.fixed <- clgeo_CollectionReport(sp.fixed)
clgeo_SummaryReport(report.fixed)
```

**Comment by eblondel** [comment]
The fix has been released in cleangeo ``0.1-1`` and now in CRAN. Closing the ticket.
